src/lms-ds-loader/lms_ds_loader/arguments.py

```diff
@@ -71,9 +47,18 @@ def set_connection_string_using_integrated_security(self, server, port, db_name)
         db_name : str
             Database name.
         """
+        assert type(server) is str, "Argument `server` must be a string"

```

Comment:
FYI, `assert isinstance(server, str)` also works.  Seems to be community debate on which is better for primitive types.

Reply:
If that works, might be better to use the `isinstance` just for consistency. Easy enough for me to try since I have these all unit tested 😀 .

Reply:
All tests pass, as expected, with change to `isinstance`.